src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -487,7 +487,13 @@ private function registerFragmentsConfiguration(array $config, ContainerBuilder
 
             return;
         }
+        $container->setParameter('fragment.renderer.hinclude.global_template', $config['hinclude_default_template']);
 
+        if (!$container->hasParameter('templating.engines') || ['twig'] == $container->getParameter('templating.engines')) {
+            $container->getDefinition('fragment.renderer.hinclude')
+                ->addTag('kernel.fragment_renderer', ['alias' => 'hinclude'])
+                ->replaceArgument(0, new Reference('twig'));
```

Comment:
FramewokrBundle injecting a reference to `twig` is wrong, as it does not depend on TwigBundle. That code should stay where it was in TwigBundle IMO.